recipes/libpng/all/conanfile.py

```diff
@@ -174,6 +185,9 @@ def package_info(self):
 
         prefix = "lib" if self._is_msvc else ""
         suffix = "d" if self.settings.build_type == "Debug" else ""
-        self.cpp_info.libs = ["{}png16{}".format(prefix, suffix)]
+        version_list = self.version.split('.')
+        major_min_version = "{}{}".format(version_list[0],version_list[1])
```

Comment:
```suggestion
        major_min_version = "{}{}".format(tools.Version(self.version).major, tools.Version(self.version).minor)
```
https://docs.conan.io/en/latest/reference/tools.html#tools-version